src/event-helper-listen.js

```diff
@@ -41,11 +43,34 @@ export function internalListenImplementation(element, eventType, listener,
       throw e;
     }
   };
+
+  // Test whether browser supports the passive option or not
+  let passiveSupported = false;
+  try {
```

Comment:
couple of unit tests to make sure detection is works as browser's implementation may change. Ideally tests call the top-level exported `listen`, mock the `addEventListener` for `test-passive` to either support or not support and then spy on `addEventListener` and `removeEventListener` to make sure the third arg is correct